Library Genesis.js

```diff
@@ -0,0 +1,268 @@
+{
+	"translatorID": "11614156-f421-4e89-8ce0-a5e69ce3ebed",
+	"label": "Library Genesis",
+	"creator": "Reverend Wicks Cherrycoke",
+	"target": "^https?://(libgen\\.io|lib\\.gen\\.rus\\.ec)/",
+	"minVersion": "3.0",
+	"maxVersion": "",
+	"priority": 100,
+	"inRepository": true,
+	"translatorType": 4,
+	"browserSupport": "gcsi",
+	"lastUpdated": "2016-05-06 13:15:28"
+}
+
+/*
+***** BEGIN LICENSE BLOCK *****
+
+	Copyright © 2016 Rev. Wicks Cherrycoke
+	This file is part of Zotero.
+
+	Zotero is free software: you can redistribute it and/or modify
+	it under the terms of the GNU Affero General Public License as published by
+	the Free Software Foundation, either version 3 of the License, or
+	(at your option) any later version.
+
+	Zotero is distributed in the hope that it will be useful,
+	but WITHOUT ANY WARRANTY; without even the implied warranty of
+	MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+	GNU Affero General Public License for more details.
+
+	You should have received a copy of the GNU Affero General Public License
+	along with Zotero. If not, see <http://www.gnu.org/licenses/>.
+
+***** END LICENSE BLOCK *****
+*/
+
+var MIME_TYPES = {
+	'pdf': 'application/pdf',
+	'epub': 'application/epub+zip',
+	'mobi': 'application/x-mobipocket-ebook',
+	'djvu': 'image/vnd.djvu'
+};
+var MD5_REGEX = /md5=([0-9a-fA-F]+)/;
+var AUTHOR_REGEX = /author\s*=\s*{(.*?)}/;
+
+
+function detectWeb(doc, url) {
+	if (url.indexOf('book/index.php') != -1) {
+	return 'book';
+	}
+}
+
+function doWeb(doc, url) {
+	var md5Hash = MD5_REGEX.exec(url)[1];
+
+	// To save some work, we use the provided bibtex file to retrieve the
+	// metadata and use Zotero's built-in bibtex importer
+	var bibtexUrl = "/book/bibtex.php?md5=" + md5Hash;
+	ZU.processDocuments(bibtexUrl, function(bibtexDoc) {
```

Comment:
indenting is still not quite right here -- everything that's running inside this should be indented.